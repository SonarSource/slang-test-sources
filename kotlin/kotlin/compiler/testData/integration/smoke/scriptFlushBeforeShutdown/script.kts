print("A")
println("B")
print("C")
