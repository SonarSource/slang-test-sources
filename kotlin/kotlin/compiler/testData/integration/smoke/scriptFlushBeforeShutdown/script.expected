OUT:
AB
C
Return code: 0
